src/utils/messageService.js

```diff
@@ -0,0 +1,107 @@
+import AffinidiDidAuthService from "@affinidi/affinidi-did-auth-lib/dist/DidAuthService/DidAuthService";
+import config from "../config";
+
+const MESSAGE_SERVICE_BASE_URL = "http://localhost:3000/api/v1";
```

Comment:
@Yengas lets have it as process.env var as well, and as default use localhost for example here